packages/number-input/src/NumberInput/NumberInput.tsx

```diff
@@ -142,16 +144,17 @@ export const NumberInput = React.forwardRef<HTMLInputElement, NumberInputProps>(
               />
             )}
           </div>
-          <div
-            className={errorMessageWrapperStyles}
-            aria-live="polite"
-            aria-relevant="all"
-            id={errorMessageId}
-          >
-            {renderErrorMessage && (
-              <Error className={errorMessageStyles}>{errorMessage}</Error>
-            )}
-          </div>
+          {showError && (
+            <div
+              id={errorMessageId}
+              className={errorContainerStyles}
+              aria-live="polite"
```

Comment:
I believe that this div was outside of the conditional check because of `aria-live`. Can you check that the error message is still announced on VoiceOver?

Reply:
Tested and confirmed that it wasn't announcing as the errorMessage rendered. I moved the conditional 1 node down

Reply:
Does `FormField` use this pattern? if not, we should consider adding it.